### PR TITLE
Use actual acronym to generate ID of barrel regions

### PR DESCRIPTION
**Pull request by lecriste**
The previous code is computing the barrel region ID from a string different from the region acronym.

### CODE REVIEW

File: atlas_splitter/barrel_splitter/somatosensory_barrels.py [hunk 1]
```diff
@@ -41,10 +39,10 @@ def layer_ids(
     new_ids: Dict[str, Dict[str, int]] = {}
     for name in names:
         new_ids[name] = {}
-        new_ids[name][name] = id_from_acronym(region_map, name)
```
Comment (eleftherioszisis) [inline atlas_splitter/barrel_splitter/somatosensory_barrels.py:44]: Why are you returning a negative id?

Comment (lecriste) [reply atlas_splitter/barrel_splitter/somatosensory_barrels.py:44]: It's just the best `int` placeholder I could think of.

File: atlas_splitter/barrel_splitter/somatosensory_barrels.py [hunk 2]
```diff
@@ -25,13 +26,10 @@
 HierarchyDict = Dict[str, Any]
 
 
-def layer_ids(
-    region_map: RegionMap, names: List[str], layers: List[str]
-) -> Dict[str, Dict[str, int]]:
+def layer_ids(region, names: List[str], layers: List[str], region_map) -> Dict[str, Dict[str, int]]:
```
Comment (lecriste) [inline atlas_splitter/barrel_splitter/somatosensory_barrels.py:29]: Does the docstring need to be updated?

File: atlas_splitter/barrel_splitter/somatosensory_barrels.py [hunk 3]
```diff
@@ -147,13 +144,14 @@ def add_hierarchy_child(
 
 
 def edit_hierarchy(  # pylint: disable=too-many-arguments
+    region: str,
```
Comment (lecriste) [inline atlas_splitter/barrel_splitter/somatosensory_barrels.py:147]: Does the docstring need to be updated?

File: atlas_splitter/barrel_splitter/somatosensory_barrels.py (hunk 2 above)
Comment (eleftherioszisis) [reply atlas_splitter/barrel_splitter/somatosensory_barrels.py]: Indeed. Updated.

File: atlas_splitter/barrel_splitter/somatosensory_barrels.py (hunk 3 above)
Comment (eleftherioszisis) [reply atlas_splitter/barrel_splitter/somatosensory_barrels.py:150]: Indeed. Updated.

**Review by eleftherioszisis — APPROVED** [review]

**Pull request merged**